File: week3 - Application Development 1 CRUD with Cassandra/crud-nodejs/update.js
const connection = require('./db_connection')

// this is a update statement in nodejs
connection.client.execute(
    'UPDATE killrvideo.user_credentials SET password = ? WHERE email = ?',
    ['Cr1st1n@sN3wP@ssW0rd', 'cv@datastax.com'],
    { prepare : true }
)
.then(function (result){
    console.log('Success')
    connection.client.shutdown()
})
.catch(function (error){
     console.log(error.message)
     connection.client.shutdown()
});

File: week3-AppDev-crud/crud-python/update.py
#!/usr/bin/env python3
from db_connection import Connection

# this is a update statement in python
try:
    connection = Connection()
    connection.session.execute(
        "UPDATE killrvideo.user_credentials SET password = %s WHERE email = %s",
        ['Cr1st1n@sN3wP@ssW0rd', 'cv@datastax.com']
    )
except:
    print('Failure')
else:
    print('Success')
finally:
    connection.close()